Classes/Public/Constants/HTMLConstants.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+public enum HTMLTags: String {
+    case Bold = "b"
+    case Delete = "del"
+    case Emphasized = "em"
+    case Italic = "i"
+    case Link = "a"
+    case StrikeThrough1 = "s"
+    case StrikeThrough2 = "strike"
```

Comment:
Considering [there is a `<link>` tag](http://www.w3schools.com/tags/tag_link.asp), I think yours is a good idea.

As for the enum names, I was a bit torn between Swift 2 (first character upper-case) and Swift 3 style (first character lower-case).

The reason I decided to go for upper case is that Swift 3 is not here yet (and even if unlikely there's a chance that rule may never see the light of day?).